user-interface/6001-SORD-submit_orders.md

```diff
@@ -145,4 +145,4 @@ Market in continuous trading:
 
 Market in auction:
 - Attempt Market order in Auction mode: should warn order is invalid
-- Attempt limit order GFN when market is normally Continuos, should warn
\ No newline at end of file
+- Attempt limit order GFN when market is normally Continuos, should warn
```

Comment:
```suggestion
- Attempt limit order GFN when market is normally Continuous, should warn
```
What's the warning here, can't GFN be placed in normal?

Reply:
spelling fixed : https://github.com/vegaprotocol/specs/pull/1181/commits/05f73b451906233c1adfa206ca5fa35cbc0c4451

added a clarification here: https://github.com/vegaprotocol/specs/pull/1181/commits/81c1f6a7ac267f18ad590f6d302b5cebba67172b